README.md

```diff
@@ -143,12 +100,8 @@ func main() {
 
 **Observation 2:** the line beginning with "`Opts :=`" sets up the options for
 `Connect()`. In this example, there is only one thing in the structure, a user
-name. The structure can also contain:
-
-* `Pass` (password),
-* `Timeout` (maximum number of milliseconds to wait before giving up),
-* `Reconnect` (number of seconds to wait before retrying if a connection fails),
-* `MaxReconnect` (maximum number of times to retry).
+name. The structure can also contain other settings, see more in
+[documentation][godoc-opts-url] for `Opts` structure.
 
```

Comment:
**Observation 1:** The line starting with "`Opts :=`" sets up the options for
`Connect()`. In this example, the structure contains only a single value, the username. The structure may also contain other settings, see more in
[documentation][godoc-opts-url] for the `Opts` structure.

Reply:
Updated. (it is actually an observation number 2, not 1)